index.html

```diff
@@ -3,184 +3,196 @@
     </div>
 </div>
 
-<div class="container spaced" id="homepage-sponsors">
-    <div class="row">
-        <div class="col-12">
-            <a href="/about/sponsors.html">
-                <h2 class="header">Sponsors</h2>
-            </a>
-        </div>
-        <div class="col-4 col-lg-2" id="sponsor-1">
-            <a href="https://rit.edu/">
-            <img src="https://assets.csh.rit.edu/pubsite/sponsors/ritLarge.png">
-            </a>
-        </div>
-        <div class="col-4 col-lg-2" id="sponsor-2">
-            <a href="https://google.com/">
-                <img src="https://assets.csh.rit.edu/pubsite/sponsors/google.png">
-            </a>
-        </div>
-        <div class="col-4 col-lg-2" id="sponsor-3">
-            <a href="https://microsoft.com/">
-            <img src="https://assets.csh.rit.edu/pubsite/sponsors/microsoft.png">
-            </a>
-        </div>
-        <div class="col-4 col-lg-2" id="sponsor-4">
-            <a href="https://slack.com/">
-            <img src="https://assets.csh.rit.edu/pubsite/sponsors/slack.png">
-            </a>
-        </div>
-        <div class="col-4 col-lg-2" id="sponsor-5">
-            <a href="https://hp.com">
-            <img src="https://assets.csh.rit.edu/pubsite/sponsors/hp.png">
-            </a>
-        </div>
-        <div class="col-4 col-lg-2" id="sponsor-6">
-            <a href="https://oracle.com/">
-            <img src="https://assets.csh.rit.edu/pubsite/sponsors/oracle.png">
-            </a>
+<!-- Twitter embed -->
+
+<div style="background-color: #b0197e;" id="twitter_embed">
+    <h3 style="color: white; padding-top: 1em; margin-top: 0em;" class="left_offset">See what we've been up to ...</h3>
+    <div class="centered" style="padding-bottom: 1em; max-width: 90%; margin: auto;">
+        <a class="twitter-timeline" data-width="600" data-height="800" href="https://twitter.com/RITCSH?ref_src=twsrc%5Etfw" alt="Twitter feed. Please disable Firefox strict blocking to see this element. ">Tweets by RITCSH</a> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script> 
+    </div>
+</div>
+
+<!-- Sponsors -->
+
+<div class="width-capped" style="background-color: #f4f4f6;">
+    <h3 class="left_offset" style="padding-top: 0.5em; margin-top: 0em; margin-bottom: 0em;">With a special thanks to our sponsors ...</h3>
+    <div class="container" id="my-homepage-sponsors">
+        <img src="https://assets.csh.rit.edu/pubsite/sponsors/ritLarge.png" alt="RIT logo">
+        <img src="https://assets.csh.rit.edu/pubsite/sponsors/google.png" alt="Google logo">
+        <img src="https://assets.csh.rit.edu/pubsite/sponsors/microsoft.png" alt="Microsoft logo">
+        <img src="https://assets.csh.rit.edu/pubsite/sponsors/slack.png" alt="Slack logo">
+        <img src="https://assets.csh.rit.edu/pubsite/sponsors/hp.png" alt="HP Logo">
+        <img src="https://assets.csh.rit.edu/pubsite/sponsors/oracle.png" alt="Oracle">
+    </div>
```

Comment:
The current pubsite shuffles the sponsors list, are you dropping that functionality?

Reply:
Yes. I decided to drop that for the time being to simplify things, then reintroduce the functionality when I have the chance to revisit the project.